source/reference/api/resume.txt

```diff
@@ -0,0 +1,65 @@
+.. _c2c-api-resume:
+
+==========
+``resume``
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Description
+-----------
+
+Resumes a paused synchronization session based on data stored on the
+destination cluster.
+
+Requirement
+-----------
+
+To use the ``resume`` endpoint, the {+c2c-product-name+} must be in the
+``PAUSED`` state.
+
+Request
+-------
+
+.. code-block:: http
+
+   POST /api/v1/resume 
+
+Request Body Parameters
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This endpoint does not use HTTP request body parameters.
```

Comment:
The request body MUST have the --data parameter and the empty `{}`, otherwise it will throw an non-fatal error

e.g.
curl localhost:27182/api/v1/pause -XPOST --data '{}'
curl localhost:27182/api/v1/resume -XPOST --data '{}'

we should fix this in pause as well

Reply:
Updated